docs/deepsemgrep/deepsemgrep-introduction.md

```diff
@@ -68,31 +68,22 @@ To enable Semgrep Pro Engine in Semgrep Cloud Platform, follow these steps:
 1. Sign in to [Semgrep Cloud Platform](https://semgrep.dev/login).
 1. Select **[Settings](https://semgrep.dev/orgs/-/settings)**.
 1. Enable the <i class="fa-solid fa-toggle-large-on"></i> **Semgrep Pro Engine beta** toggle.
-1. Ensure that you have the **default ruleset** enabled in your **[Rule Board](https://semgrep.dev/orgs/-/board)**. This ruleset can be found in the **Monitor column**. If this ruleset is **not** added, go to [https://semgrep.dev/p/default](https://semgrep.dev/p/default) and click **Add to Rule Board**.
-
-### Testing through a demo project
-
-After enabling Semgrep Pro Engine, you can add a demo project to quickly try Semgrep Pro Engine.
-
-<AddDemoProject />
-
-You have now successfully run Semgrep Pro Engine and Pro rules on a project.
+1. Ensure that you have the **default ruleset** enabled in your **[Rule Board](https://semgrep.dev/orgs/-/board)**. In the Rule Board, the **default ruleset** is in the **Monitor column**. If this ruleset is **not** added, go to [https://semgrep.dev/p/default](https://semgrep.dev/p/default), and then click **Add to Rule Board**.
+1. Optional: If you don't have any projects added in your organization, follow the procedures described in [Scanning a repository from GitHub or GitLab](/semgrep-code/getting-started-with-semgrep-code/#semgrep-code-with-semgrep-cloud-platform) to scan a new project with Semgrep Pro Engine.
 
 ## Additional information
 
 ### Experimental support for interprocedural analysis
 
-Interprocedural analysis keeps the fast scan times of Semgrep OSS while also finding new vulnerabilities that cross functions. It's designed to improve results shown in pull requests and help security engineers build trust with developers.
+Interprocedural analysis keeps the fast scan times of Semgrep OSS while also finding new vulnerabilities that cross functions. It's designed to improve results of pull or merge requests (PRs or MRs) and help security engineers build trust with developers.
```

Comment:
I would revert the change. "shown in" is correct. The result of the semgrep scan is displayed to the users. NOT the result of a pull or merge request.

Reply:
```suggestion
Interprocedural analysis keeps the fast scan times of Semgrep OSS while also finding new vulnerabilities that cross functions. It's designed to improve results shown in pull or merge requests (PRs or MRs) and help security engineers build trust with developers.
```